firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/GenerativeModel.kt

```diff
@@ -171,19 +160,19 @@ internal constructor(
     generateContent(content { image(prompt) })
 
   /**
-   * Generates a streaming response from the backend with the provided bitmap represented [Content].
+   * Generates a streaming response from the backend with the provided bitmap prompt.
    *
    * @param prompt The bitmap to be converted into a single piece of [Content] to send to the model.
    * @return A [Flow] which will emit responses as they are returned from the model.
    */
   fun generateContentStream(prompt: Bitmap): Flow<GenerateContentResponse> =
     generateContentStream(content { image(prompt) })
 
-  /** Creates a chat instance which internally tracks the ongoing conversation with the model */
+  /** Creates a [Chat] instance which internally tracks the ongoing conversation with the model */
   fun startChat(history: List<Content> = emptyList()): Chat = Chat(this, history.toMutableList())
 
   /**
-   * Counts the number of tokens used in a prompt.
+   * Counts the number of tokens in a prompt.
```

Comment:
Nit
```suggestion
   * Counts the amount of tokens in a prompt.
```

If we adopt this, should also do it below

Reply:
Done here and elsewhere.